app/helpers/gtl_helper.rb

```diff
@@ -0,0 +1,125 @@
+module GtlHelper
+  def gtl_selected_records
+    records = @edit[:object_ids] unless @edit.nil? || @edit[:object_ids].nil?
```

Comment:
nitpick1: consider using try syntax instead
nitpick2: can you use ||= on following lines to it doesn't even get evaluated?

```ruby
records = @edit.try(:[], :object_ids)
recors ||= ....
```

Reply:
I did not refactor this bit this time. I just moved the code.

Reply:
But yes, you are right. But this is gaprindashvili stuff and we don't want to refactor too much.

Reply:
I don't see that as a high risk? i think it make sense to clean up if you are here already.

Reply:
Well if I did as you wrote I would change the behavior of the method (and possibly create a regression). The current priority of the variables is the other way around. `:object_ids` have the lowest priority.

So it's a simple change but is it safe? No, that's not that simple.

But ok, doing it anyway ;-)

Reply:
aaand if `@edit` is `nil` I have one more regression

Reply:
> aaand if `@edit` is `nil` I have one more regression

The above is actually not true. So just the order of the variables would be wrong ;-)